README.md

```diff
@@ -205,6 +223,15 @@ The Lock/Unlock trait is used for anything that can lock and unlock, such as doo
 - Lock Command: A device command used to lock the device.  Should not require any parameters.  Maps to `lock` by default.
 - Lock Command: A device command used to unlock the device.  Should not require any parameters.  Maps to `unlock` by default.
 
+### Media State
+
+The Media State trait is used for reporting the current playback and activity state of a media device.  Query command is unknown at the is time.  The trait has the following configuration parameters:
+
+- Support Activity State: Should be set if this device can report the current activity state.  Defaults to 'false'.
+- Support Playback State: Should be set if this device can report the current playback state.  Defaults to 'false'.
+- Activity State Attribute: The device attribute used to query the current activity state of the device.  Maps to the `activityState` attribute by default.
```

Comment:
This setting should only be available if "Support Activity State" is true.  It should probably have no default, since there's no matching Hubitat capability.

Reply:
Removed non-matching defaults.